migration/sql-files/022-invitations.sql

```diff
@@ -0,0 +1,35 @@
+create table invitation (
+  invitation_id uuid primary key DEFAULT uuid_generate_v4(),
+  invite_to uuid NOT NULL references identities (id),
+  user_id uuid NOT NULL references identities (id),
+  member boolean NOT NULL,
+  created_at timestamp with time zone,
+  updated_at timestamp with time zone,
+  deleted_at timestamp with time zone
+);
+
+create table invitation_role (
+  invitation_id uuid NOT NULL references invitation (invitation_id),
+  role_id uuid NOT NULL references role (role_id)
+  PRIMARY KEY (invitation_id, role_id)
```

Comment:
Could we have an ID column which is an auto-generated uuid - and then make that the primary key.
We can make `(invitation_id, role_id)` a unique key.

Apart from general consistency with all other tables, this ensures that the database row cleanup logic doesn't break  https://github.com/fabric8-services/fabric8-auth/blob/master/gormsupport/cleaner/db_clean.go 
This helps the test data cleanup to happen 'automatically' using the primary key of the entity.

We don't need to use the ID anywhere in the code - apart from the repository struct.